ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-ref.yaml

```diff
@@ -13,8 +13,22 @@ ref:
   - namespace: test-credentials
     name: nutanix-pc
     mount_path: /var/run/vault/nutanix
+  env:
+  - name: SIZE_VARIANT
+    default: ""
+    documentation: |-
+      The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.
+
+      The sizes are:
+      * "" (default) - 4 vCPU, 16GB control plane nodes, default workers
+      * "compact" - 8 vCPU, 32GB control plane nodes, no workers
```

Comment:
what's the default value of cpu/memory on control plane nodes? does it need to adjust them when `SIZE_VARIAN` set to `compact`?

Reply:
Yeah, agree. Generally compact cluster need to adjust the default cpu/memory setting so that the cluster has enough resource to run containers.